docs/LV_client/shared_libraries/mono_light_API_example.md

```diff
@@ -0,0 +1,56 @@
+# mono_light_API_example.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_light_API_example.png"/>
+</p>
+
+### Description
+
+Enables users to control the vehicle's steering, throttle, brakes, starting position, and weather simulation as well . This example enables the user to configure and update the multiple settings for lights in the ego vehicle.
```

Comment:
Looks like there may have been more to the first sentence, but was removed. You do not need "as well" at the end of this sentence anymore. Depending on the use of "multiple settings" remove the "the" in front. If it's an element that users are clicking on or specifically called "multiple settings", keep the "the" otherwise you are referencing it as a broad term for many settings, remove the "the."